src/data/store.ts

```diff
@@ -340,7 +340,7 @@ async function init() {
     const playlistsDir = await audioDir();
     if (!fileSettings.playlistsLocation) {
         fileSettings.playlistsLocation =
-            playlistsDir.concat("/Musicat Playlists");
+            path.join(playlistsDir, "Musicat Playlists");
```

Comment:
@basharovV It's missing an `await`. I've created https://github.com/basharovV/musicat/pull/42 to fix the regression. Sorry ;)